### PR TITLE
Issue 181 netezza support

### LINKED ISSUE #181: Add Netezza support to Execute Engine
Now ExecutionEngine doesn't support Netezza

image-2019-03-29-17-39-30-176

[ee.log](https://github.com/OHDSI/ArachneExecutionEngine/files/3029538/ee.log)

Need to add it

### PR DESCRIPTION
**Pull request by ssuvorov-fls**
fixes #181

### CODE REVIEW

**Comment by ssuvorov-fls** [comment]
R uses RODBC to connect to Netezza. This driver is already used in R code

**Comment by pavgra** [comment]
Where is the `RODBC` used by `DatabaseConnector`?
image

image


Attempt to execute PLP gives expected exception:
```

  
v  checking for file '/tmp/Rtmp32eSbD/file4b0664ad20e/analysis_1/DESCRIPTION'

  
-  preparing 'PredictionAnalysis.o9i84bu1':
v  checking DESCRIPTION meta-information

  
-  checking for LF line-endings in source and make files and shell scripts

  
-  checking for empty or unneeded directories
   Removed empty directory 'PredictionAnalysis.o9i84bu1/inst/shiny/PLPViewer/data'
-  building 'PredictionAnalysis.o9i84bu1_0.0.2.tar.gz'

  
   

Installing package into '/usr/local/lib/R/site-library'
(as 'lib' is unspecified)
* installing *source* package 'PredictionAnalysis.o9i84bu1' ...
** R
** inst
** preparing package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded
* DONE (PredictionAnalysis.o9i84bu1)
Creating cohorts
Connecting using Netezza driver
Error in findPathToJar("^nzjdbc\\.jar$", pathToDriver) : 
  No drives matching pattern ^nzjdbc\.jar$ found in folder /usr/local/lib/R/site-library/DatabaseConnectorJars/java. Please download the JDBC driver, then add the argument 'pathToDriver', pointing to the local path to directory containing the JDBC JAR file. Type ?jdbcDrivers for help on downloading drivers.
Calls: tryCatch ... createCohorts -> <Anonymous> -> connect -> findPathToJar
Removing package from '/usr/local/lib/R/site-library'
(as 'lib' is unspecified)
```

**Comment by ssuvorov-fls** [comment]
netezza drivers added for R execution

File: pom.xml [hunk 1]
```diff
@@ -511,6 +518,44 @@
...
+                                <phase>initialize</phase>
+                                <goals>
+                                    <goal>install-file</goal>
+                                </goals>
+                                <configuration>
+                                    <groupId>org.netezza</groupId>
+                                    <artifactId>nzjdbc</artifactId>
+                                    <version>1.0</version>
+                                    <packaging>jar</packaging>
+                                    <file>${basedir}/src/main/extras/netezza/nzjdbc.jar</file>
```
Comment (pavgra) [inline pom.xml]: Let's be consistent and apply the same approach for similar functional blocks: specifically, for the BQ I can pass drivers location in a parameter (`bigquery.classpath`)

Comment (ssuvorov-fls) [reply pom.xml]: fixed

**Pull request merged**